### PR TITLE
[#PAB-287] fix multiple ingest calls

### PR DESCRIPTION
**Pull request by cyrusdobbs**
https://trello.com/c/wGt8Jv9L/287-bug-multiple-calls-to-ingest-endpoint-causes-internal-error

### Change description
Fixes multiple calls to ingest and adds an integration test

- [X] Unit tests and other appropriate tests added or updated
- [NA] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: core/controllers/ingest.py [hunk 1]
```diff
@@ -43,6 +43,10 @@ def ingest(body):
     if validation_failures:
         raise ValidationError(validation_failures=validation_failures)
 
+    # wipe db
+    db.drop_all()
+    db.create_all()
```
Comment (gidsg) [inline core/controllers/ingest.py:48]: This seems quite dangerous, can we add a TODO to remove this before we go live?

File: tests/controller_tests/conftest.py [hunk 2]
```diff
@@ -9,4 +9,5 @@
 @pytest.fixture(scope="function")
 def example_file() -> BinaryIO:
     """An example spreadsheet in towns-fund schema format."""
-    return (resources / "DLUCH_Data_Model_V3.4_EXAMPLE.xlsx").open("rb")
+    with open(resources / "DLUCH_Data_Model_V3.4_EXAMPLE.xlsx", "rb") as file:
```
Comment (nsnape) [inline tests/controller_tests/conftest.py:12]: nice. Should we swap the other tests that open files to use context manager too? (in test_ingest). Not really part of this PR, but could add here anyway? Or just backlog it as a minor fix and do it later?

**Review by nsnape — APPROVED** [review]

File: tests/controller_tests/conftest.py (hunk 2 above)
Comment (cyrusdobbs) [reply tests/controller_tests/conftest.py:12]: happy to add it now 👍

File: core/controllers/ingest.py (hunk 1 above)
Comment (cyrusdobbs) [reply core/controllers/ingest.py:49]: added this

**Review by gidsg — APPROVED** [review]

**Pull request merged**